brigade/static/scss/molecules/_card.scss

```diff
@@ -5,32 +5,119 @@
 //
 
 .card {
+  font-weight: inherit;
+
+  display: flex;
+  flex-direction: row;
+
+  margin: 15px 0;
+
   box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
+  flex-wrap: wrap;
+
+  /* Make the parts of each card full-width by default */
+  & > * {
+    flex: 100% 1;
+  }
+
+  &:hover {
+    box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
+  }
+
+  /* Make sure the first element in each part of the card doesn't have a top margin */
+  & > * > :first-child {
```

Comment:
This feels quite invasive to me, that if we try to put other components inside a card, that this rule could override their styling. Probably okay for now since I imagine you're meaning to target the margin-top on `<p>` tags, but still something I argue we should generally avoid.